client/src/components/organisms/Modal.tsx

```diff
@@ -0,0 +1,30 @@
+import '../../pages/Modal.scss';
+import React, { useState } from 'react';
+import { useApolloClient } from "@apollo/client";
+import { useDispatch } from "react-redux";
```

Comment:
Don't think we need to import the stuff in the curly braces since they're not used in the rest of the component